packages/serverpod/lib/src/server/health_check.dart

```diff
@@ -37,45 +39,48 @@ Future<ServerHealthResult> defaultHealthCheckMetrics(
 
   // Check database response time
   var dbResponseTime = 0.0;
-  var dbHealthy = false;
+  bool? dbHealthy;
 
-  try {
-    var startTime = DateTime.now();
-    var rnd = Random().nextInt(1000000);
+  if (FeatureDecisions.enableDatabase) {
```

Comment:
Not sure, but should `enableScheduledHealthChecks` be used for this? Or, maybe `&&` with `enableDatabase`

Reply:
My thinking here was that you could still perform a health check here but since we do not support the database we would opt out of the db check (because that feature is disabled). The ScheduledHealthCheck is a different thing that runs on a timer.